File: Backend/API.Facturacion/DTO/Clases/Factura.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Facturacion.DTO.Clases
{
    public class Factura
    {
        public int Id { get; set; }
        public Cliente Cliente { get; set; }
        public string Numero { get; set; }
        public DateTime FechaExpedicion { get; set; }
        public List<FacturaDetalle> ListaFacturaDetalles { get; set; }
        public decimal ValorTotal { get; set; }
    }
}

File: Frontend/src/app/models/factura.ts
import { Cliente } from "./cliente";
import { FacturaDetalle } from "./factura-detalle";

export class Factura {
    id: number;
    cliente: Cliente;
    fechaExpedicion: Date;
    numero: string;
    listaFacturaDetalles: FacturaDetalle[];
    valorTotal:number;
}
